src/sentry/api/serializers/models/commit.py

```diff
@@ -1,18 +1,61 @@
 from __future__ import absolute_import
 
-from sentry.api.serializers import Serializer, register
-from sentry.models import Commit
+from sentry.api.serializers import Serializer, register, serialize
+from sentry.db.models.query import in_iexact
+from sentry.models import Commit, Repository, UserEmail, User
 
 
 @register(Commit)
 class CommitSerializer(Serializer):
+    def get_attrs(self, item_list, user):
+
+        authors = set(c.author for c in item_list if c.author is not None)
+
+        user_emails = UserEmail.objects.filter(
```

Comment:
This code looks really similar to what we did in `ReleaseSerializer`. Can we turn some of this into a helper function?